src/_tutorials/server/cmdline.md

```diff
@@ -430,27 +372,24 @@ The dart:convert library contains these and other data converters,
 including one for JSON.
 To use these converters you need to import the dart:convert library:
 
-{% prettify dart %}
+<?code-excerpt "misc/bin/dcat.dart" retain="dart:convert"?>
+```dart
 import 'dart:convert';
-{% endprettify %}
+```
 
 ## Writing a file
 
 The easiest way to write text to a file is to create a
-<a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/File-class.html" target="_blank">File</a>
+[File]({{ioAPI}}/File-class.html){:target="_blank"}
 object and use the `writeAsString()` method:
 
-{% prettify dart %}
-File quotesFile = new File('quotes.txt');
-String stronger = 'That which does not kill us makes us stronger. -Nietzsche';
-
-try {
-  await quotesFile.writeAsString(stronger, mode: FileMode.append);
-  print('Data written.');
-} catch (e) {
-  print('Oops!');
-}
-{% endprettify %}
+<?code-excerpt "misc/lib/tutorial/cmdline.dart (write quote)"?>
+```dart
+final quotes = File('quotes.txt');
+const stronger = 'That which does not kill us makes us stronger. -Nietzsche';
+
+await quotes.writeAsString(stronger, mode: FileMode.append);
+```
```

Comment:
I dropped the try-catch statements from this code excerpt for a couple of reasons: (1) IMHO, it adds nothing to the illustrated use of `writeAsString()`. (2) To be consistent with the next code excerpt (illustrating `openWrite()` and `write()`), which doesn't use try-catch.

Reply:
Makes me slightly nervous, but I trust your judgment.